Assets/Scripts/SmallHazardDestroyerScript.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SmallHazardDestroyerScript : MonoBehaviour
+{
+    private void OnCollisionEnter(Collision other) 
+    {
+        switch (other.gameObject.tag)
```

Comment:
Is this switch statement even necessary if all we are doing is Destroying the game object on collision? This could just be a 1 liner. 

could simply just replace the whole thing with Destroy(gameObject);

Is there any condition where we won't destroy the small projectile hazard on collision? I think that's probably the behavior we want though.

Reply:
> I mean we only want it to be destroyed if it collides with bullets or the player, if it collides with other hazards we don't want it to be destroyed

Reply:
If you are going to keep the switch statement then you need to add a default case. If you don't then we will have projectiles bouncing off of walls and other objects. 

Probably they should just be destroyed on contact with any collider. If this is the case, I would just make this a simple one liner and replace the entire switch statement with `Destroy(gameObject)` If at some point we need different behavior for some condition we can go back and make this a conditional.